### PR TITLE
site: add dividers between documentation links and remove the list item bullets

### PR DESCRIPTION
**Pull request by sophiapoirier**
Aside from the work-in-progress museum, the only other thing on our nouveau website that I don't like as much is the documentation list.  I think the look is a bit heavy with tightly spaced large-bold-font links with bullets.  This page used to be a table with last-update dates, the latter of which I don't really think is necessary, but I played around a bit with making a little more space and using that to add some visual delineation between the links.  like / dislike / alterations?

### CODE REVIEW

File: site/docs.html [hunk 1]
```diff
@@ -12,6 +12,19 @@
    }
    ul.doclist {
      font-size: 18px;
+     list-style-type: none;
+     padding-left: 1em;
+     padding-right: 1em;
+   }
+   li {
+     border-top: 1px solid #ddd;
+     padding: 0.12em;
```
Comment (tom7) [inline site/docs.html:21]: I think it looks good as-is, though I'd recommend
  padding: 0.2em 0;

... to add a smidge more padding (the parentheses in this font on Windows seem to descend pretty far). 0 horizontal padding to keep the dividers aligned with the text for a clean left edge at least. If you like it the way it is, keep it!

File: site/docs.html [hunk 2]
```diff
@@ -12,6 +12,19 @@
...
+   }
+   li.first {
```
Comment (tom7) [inline site/docs.html:23]: IMO better to do li:first-child here, and then you can leave off class="first" below. This keeps the styling separate from the list structure.

**Review by tom7 — APPROVED** [review]
Thanks! Looks good with some optional suggestions.

Will get back to the DFX stuff soon but no need to hold this up.

File: site/docs.html (hunk 1 above)
Comment (sophiapoirier) [reply site/docs.html:21]: yup agreed each of those suggestions is an improvement!

File: site/docs.html (hunk 2 above)
Comment (sophiapoirier) [reply site/docs.html:23]: oh very good, I did not know about that.

**Pull request merged**